health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/ViewHierarchy.js

```
@@ -397,7 +397,7 @@ const ViewHierarchy = () => {
                                         return (
                                             <div>
                                                 <div className="hierarchy-boundary-sub-heading2">
-                                                    {`${t(( hierarchyType + "_" + hierItem?.boundaryType).toUpperCase())}`}
+                                                    {`${t(( hierarchyType + "_" + hierItem?.boundaryType.trim().replace(/[\s_]+/g, '')).toUpperCase())}`}
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)